tests/core/interfaces-gpio-chardev/task.yaml

```diff
@@ -29,15 +29,22 @@ prepare: |
     systemctl enable --now gpio-sim.service
     tests.cleanup defer systemctl disable --now gpio-sim.service
 
+    #shellcheck disable=SC2010
+    sysfs_gpiochip_num="$(ls /sys/class/gpio/ | grep gpiochip | head -n1 | cut -c9-)"
```

Comment:
iirc the connection should still happen if the exporter service fails, so no need to above and beyond to get it correct, but it's nice touch